packages/server/src/fhir/repo.test.ts

```diff
@@ -172,6 +173,27 @@ describe('FHIR Repo', () => {
     expect(result3.link?.find((e) => e.relation === 'next')).toBeUndefined();
   });
 
+  test('Search previous link', async () => {
+    const family = randomUUID();
+
+    for (let i = 0; i < 2; i++) {
+      await systemRepo.createResource({
+        resourceType: 'Patient',
+        name: [{ family }],
+      });
+    }
+
+    const result1 = await systemRepo.search({
+      resourceType: 'Patient',
+      filters: [{ code: 'name', operator: Operator.EQUALS, value: family }],
+      count: 1,
+      offset: 1,
+    });
+    expect(result1.entry).toHaveLength(1);
+    expect(result1.link).toBeDefined();
+    expect(result1.link?.find((e) => e.relation === 'previous')).toBeDefined();
```

Comment:
add coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1978&selected=medplum_medplum%3Apackages%2Fserver%2Fsrc%2Ffhir%2Frepo.ts&line=1168